recompose-flow.md

```diff
@@ -200,18 +195,18 @@ const enhancer: HOC<*, EnhancedCompProps> = compose(
 )
 ```
 
-Now flow will infer type of `data` property so you can safely use it.
+Now Flow will infer the type of `data` properly, so you can safely use it.
 
 ![data-type-example](./dataExample.png?raw=true)
 
 ## Links
 
-Fully typed [example project](https://github.com/acdlite/recompose/tree/master/types/flow-example) of this [menu app](https://grader-meets-16837.netlify.com/)
+- A fully-typed [example project](https://github.com/acdlite/recompose/tree/master/types/flow-example) of this [menu app](https://grader-meets-16837.netlify.com/)
 
-Usage examples can be found in [tests](https://github.com/acdlite/recompose/tree/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-)
+- Usage examples can be found in [tests](https://github.com/acdlite/recompose/tree/master/types/flow-typed/recompose_v0.24.x/flow_v0.53.x-)
 
-## PS
+## P.S.
 
-Your turn Caleb ;-)
+Your move, Caleb ;-)
```

Comment:
I thought "move" sounded cooler, but you could change it back to "turn" if you like! "Move" implies chess, but "turn" could be any game. (I don't know why.)

Reply:
Actually Caleb already did his move :)
https://github.com/facebook/flow/commit/ab9bf44c725efd2ed6d7e1e957c5566b6eb6f688